### PR TITLE
Initial brute force attempt to get something on paper

### LINKED ISSUE #9: Support multiple TED6000 System Types
TED6000 has 3 different System Types that can be configured:
```
0 – NET
1 -  NET + GEN 
2 – NET + LOAD
``` 
I don't really know how they work since I only have one MTU and can't really see the system's behavior. My guess is they will show data across the different DashData.xml queries.

### PR DESCRIPTION
**Pull request by realumhelp**
Fixes #9 -
Simple attempt (ugly) at pulling the extra data for advanced system types /w generative capabilities. Hard to know the system's behavior without having all the necessary pieces in place for testing.

### CODE REVIEW

**Comment by realumhelp** [comment]
If you wanted to be a bit smarter you could probably pull the queries like this, however you'd have to make sure you don't call total_gen or total_load methods if you system type is NET. Seems like a good chunk of work when you could just call the queries and get empty values back.

```python
    async def update(self) -> None:
        """Fetch settings and power data from the endpoints."""
        await asyncio.gather(
            self._update_endpoint("endpoint_settings_results", ENDPOINT_URL_SETTINGS),
        )

        self._parse_mtus()
        self._parse_spyders()

        adv_system_type_queries = []
        if self.system_type != TED6000.SystemType.NET:
            """Only do these extra queries if advanced system type."""
            adv_system_type_queries.extend(
                [
                    self._update_endpoint(
                        "endpoint_solardash_results", ENDPOINT_URL_DASHBOARD, 1
                    ),
                    self._update_endpoint(
                        "endpoint_solardash_results", ENDPOINT_URL_DASHBOARD, 2
                    ),
                ]
            )
        """Fetch MTU/Spyder data results"""
        await asyncio.gather(
            self._update_endpoint("endpoint_mtu_results", ENDPOINT_URL_MTU),
            self._update_endpoint("endpoint_spyder_results", ENDPOINT_URL_SPYDER),
            self._update_endpoint(
                "endpoint_dashboard_results", ENDPOINT_URL_DASHBOARD, 0
            ),
            *adv_system_type_queries,
            *map(
                lambda mtu: self._update_endpoint(
                    "endpoint_dashboard_results",
                    ENDPOINT_URL_MTUDASHBOARD,
                    mtu.position,
                ),
                self.mtus,
            )
        )
```

File: src/tedpy/ted6000.py [hunk 1]
```diff
@@ -26,13 +27,21 @@
 class TED6000(TED):
     """Instance of TED6000."""
 
+    class SystemType(Enum):
+        """Defines the various TED6000 System Types"""
+
+        NET = 0
+        NET_GEN = 1
+        NET_LOAD = 2
+
```
Comment (rianadon) [inline src/tedpy/ted6000.py]: Let's move this to `dataclasses` to be consistent with the other enums.

**Review by rianadon — COMMENTED** [review]
This looks fine as is! It's only 2 queries; calling the queries regardless of whether they're necessary sounds fine.

File: src/tedpy/ted6000.py (hunk 1 above)
Comment (realumhelp) [reply src/tedpy/ted6000.py]: Moved.

**Review by rianadon — APPROVED** [review]

**Pull request merged**